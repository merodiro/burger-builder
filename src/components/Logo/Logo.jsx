import React from 'react'

import classes from './Logo.module.css'
import burgerLogo from '../../assets/images/burger-logo.png'

const Logo = () => (
  <div className={classes.Logo}>
    <img src={burgerLogo} alt="Burger Builder Logo" />
  </div>
)

export default Logo
