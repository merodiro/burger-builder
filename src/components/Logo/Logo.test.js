import React from 'react'
import ReactDOM from 'react-dom'
import { shallow } from 'enzyme'

import Logo from './Logo'

describe('<Logo>', () => {
  it('renders the same output', () => {
    const component = shallow(<Logo />)
    expect(component).toMatchSnapshot()
  })
})
