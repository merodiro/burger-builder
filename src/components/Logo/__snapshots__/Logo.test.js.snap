// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo> renders the same output 1`] = `
<div
  className="Logo"
>
  <img
    alt="Burger Builder Logo"
    src="burger-logo.png"
  />
</div>
`;
